irohad/validation/stateful_validator_common.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_STATEFUL_VALIDATOR_COMMON_HPP
+#define IROHA_STATEFUL_VALIDATOR_COMMON_HPP
+
+#include "common/types.hpp"
+
+namespace iroha {
+  namespace validation {
+
+    /// Type of per-transaction errors, which appeared during validation
+    /// process; contains errors themselves and transaction hashes
+    using TransactionsErrors = std::vector<
```

Comment:
The issue is that this structure will not be used as map in tx processor, since it has to iterate through all txs and push statuses to observable. Maybe change it to map when there will be a case for it?

Reply:
We can leave it this way. Hopefully if there will be a use case for map one won't forget to update it

Reply:
I've already done this, soon the P.R. will come